mora/integrations/serviceplatformen.py

```diff
@@ -157,6 +158,9 @@ def get_citizen(cpr):
 
 
 def _get_citizen_stub(cpr):
+    if not util.is_cpr_number(cpr):
```

Comment:
Move this to `get_citizen` so we do this check regardless of running the stub or not.